celsos_bookSearch/celsos_bookSearch/ViewModel/SearchViewModel.swift

```diff
@@ -6,3 +6,17 @@
 //
 
 import Foundation
+import UIKit
+
+class SearchViewModel {
+    public var searchesArray: [String] = []
+
+    func validateSearchedWord(searchBarOutlet: UISearchBar) -> Bool {
+        if let searchBarText = searchBarOutlet.text, !searchBarText.isEmpty, !searchesArray.contains(searchBarText) {
```

Comment:
Please implements validation about "many spaces" that are invalid. Like a "trim"

Reply:
What do you mean?

Reply:
Done